build.ps1

```diff
@@ -9,7 +9,7 @@ param (
     [string]$SemanticVersion = '1.0.0-zlocal',
     [string]$Branch,
     [string]$CommitSHA,
-    [string]$BuildBranch = 'cb604c2cd1b2f7f71fb574cdda4c83ddb1464cc7'
+    [string]$BuildBranch = 'sb-micro'
```

Comment:
Will switch to a commit after https://github.com/NuGet/ServerCommon/pull/204 is merged